pkg/daemon/ceph/osd/volume_test.go

```diff
@@ -260,6 +296,46 @@ var cephVolumeRAWTestResult = `{
 }
 `
 
+func TestInitializeBlockPVCResult(t *testing.T) {
+	executor := &exectest.MockExecutor{}
+	executor.MockExecuteCommandWithCombinedOutput = func(command string, args ...string) (string, error) {
+		logger.Infof("%s %v", command, args)
+		if args[1] == "ceph-volume" && args[2] == "raw" && args[3] == "prepare" && args[4] == "--bluestore" && args[7] == "--crush-device-class" {
+			return initializeBlockPVCTestResult, nil
+		} else if args[1] == "ceph-volume" && args[2] == "lvm" && args[3] == "prepare" && args[4] == "--bluestore" && args[7] == "--crush-device-class" {
+			data := getLVPath(initializeBlockPVCTestResult)
+			assert.NotNil(t, data)
+			return initializeBlockPVCTestResult, nil
+		} else if args[1] == "ceph-volume" && args[2] == "--log-path" && args[4] == "lvm" && args[5] == "prepare" && args[6] == "--bluestore" {
+			return initializeBlockPVCTestResult, nil
+		}
+
+		return "", errors.Errorf("unknown command %s %s", command, args)
+	}
+	os.Setenv(oposd.CrushDeviceClassVarName, "foo")
+	defer os.Setenv(oposd.CrushDeviceClassVarName, "")
+	context := &clusterd.Context{Executor: executor}
+	clusterInfo := &cephconfig.ClusterInfo{
+		CephVersion: cephver.CephVersion{Major: 14, Minor: 2, Extra: 8},
+	}
+	a := &OsdAgent{cluster: clusterInfo, nodeName: "sda"}
+	devices := &DeviceOsdMapping{
+		Entries: map[string]*DeviceOsdIDEntry{
+			"meta": {Data: -1, Metadata: []int{0, 1}, Config: DesiredDevice{Name: "lv"}},
+		},
+	}
+	cvLogDir = path.Join("/var/log/ceph", a.nodeName)
```

Comment:
`/var/log/ceph/ceph-volume.log` stores the output of the ceph-volume commands. Since we are mocking that command, we don't need to create this directory.